nilearn/_utils/niimg_conversions.py

```diff
@@ -62,6 +62,11 @@ def _iter_check_niimg(niimgs, ensure_ndim=None, atleast_4d=False,
 
     target_fov: tuple of affine and shape
        If specified, images are resampled to this field of view
+
+    dtype: type
```

Comment:
it's {dtype, "auto"} as a type description. "type" is something different in Python.